File: practiceForExam1/c/PROB10818.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);

    int numbers[N];
    for (int i = 0; i < N; i ++) {
        scanf("%d", &numbers[i]);
    }

    int min = numbers[0];
    int max = numbers[0];
    for (int j = 0; j < N; j ++) {
        if (numbers[j] < min) {
            min = numbers[j];
        }
        if (numbers[j] > max) {
            max = numbers[j];
        }
    }

    printf("%d %d", min, max);
}

File: practiceForExam1/java/PROB10818.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class PROB10818 {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;
    static int N;
    static int[] numbers;
    static int max = Integer.MIN_VALUE;
    static int min = Integer.MAX_VALUE;

    public static void main (String[] args) throws IOException {
        N = Integer.parseInt(br.readLine());
        st = new StringTokenizer(br.readLine());
        numbers = new int[N];
        for (int i = 0; i < N; i ++) {
            numbers[i] = Integer.parseInt(st.nextToken());
            min = Math.min(numbers[i], min);
            max = Math.max(numbers[i],  max);
        }

        System.out.println(min + " " + max);
    }
}
